browser/profiles/profile_util_unittest.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/profiles/profile_util.h"
+
+#include "base/files/file_path.h"
+#include "base/files/scoped_temp_dir.h"
+#include "brave/common/pref_names.h"
+#include "brave/components/search_engines/brave_prepopulated_engines.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/profiles/profile_manager.h"
+#include "chrome/test/base/scoped_testing_local_state.h"
```

Comment:
pls check for unused includes

Reply:
Is there any easy way to check? I likely copied this from another file and short of removing each (one by one), I'm not sure of a better way. The reverse (include what you use) already happens with our linting